src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -434,6 +440,8 @@ public float ProgressNotification
         /// <summary>
         /// Gets or sets the properties of the animation.
         /// </summary>
+        [Obsolete("Deprecated in API6, Will be removed in API9, Please use as keyword instead!")]
```

Comment:
Obsolete("Deprecated in API9, Will be removed in API11, Please use PropertyList instead")

Reply:
please update every Obsolete information